File: Examples/Features/Charts/ChartGallery/BarStackedChart.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.Charts.ChartGallery
{
    public class BarStackedChart :ExampleBase
    {
        public override void Execute(GrapeCity.Documents.Excel.Workbook workbook)
        {
            IWorksheet worksheet = workbook.Worksheets[0];
            GrapeCity.Documents.Excel.Drawing.IShape shape = worksheet.Shapes.AddChart(GrapeCity.Documents.Excel.Drawing.ChartType.BarStacked, 250, 20, 360, 230);
            worksheet.Range["A1:C4"].Value = new object[,] {
                { 103, 121, 109 },
                { 56, 94, 115 },
                { 116, 89, 99 },
                { 55, 93, 70 }             
            };
            shape.Chart.SeriesCollection.Add(worksheet.Range["A1:C4"], GrapeCity.Documents.Excel.Drawing.RowCol.Columns);
            shape.Chart.ChartTitle.Text = "Bar Stacked Chart";
            shape.Chart.Legend.Position = GrapeCity.Documents.Excel.Drawing.LegendPosition.Left;
        }
    }
}

File: Examples.VB/Features/Charts/ChartGallery/BarStackedChart.vb
Namespace Features.Charts.ChartGallery
    Public Class BarStackedChart
        Inherits ExampleBase
        Public Overrides Sub Execute(workbook As Excel.Workbook)
            Dim worksheet As IWorksheet = workbook.Worksheets(0)

            Dim shape As IShape = worksheet.Shapes.AddChart(ChartType.BarStacked, 250, 20, 360, 230)
            worksheet.Range("A1:C4").Value = {
                {103, 121, 109},
                {56, 94, 115},
                {116, 89, 99},
                {55, 93, 70}
            }

            shape.Chart.SeriesCollection.Add(worksheet.Range("A1:C4"), RowCol.Columns)
            shape.Chart.ChartTitle.Text = "Bar Stacked Chart"
            shape.Chart.Legend.Position = LegendPosition.Left
        End Sub
    End Class
End Namespace
